src/Illuminate/Database/Eloquent/Relations/BelongsToMany.php

```diff
@@ -784,7 +784,7 @@ public function findOr($id, $columns = ['*'], ?Closure $callback = null)
      * @param  mixed  $operator
      * @param  mixed  $value
      * @param  string  $boolean
-     * @return \Illuminate\Database\Eloquent\Model|static
```

Comment:
I am also not sure why this returns `|static`, in which case does it return an instance of `BelongsToMany`?